R/faster-rasters.R

```diff
@@ -46,9 +47,21 @@
 #' origStack <- stack(poly)
 #'
 #' # rasterize
-#' shpRas1 <- rasterize(shp, origStack)
-#' shpRas2 <- fastRasterize(shp, origStack)
-#' all.equal(shpRas1, shpRas2)
+#' shpRas1_character <- rasterize(shp, origStack, field = "other")
+#' shpRas1_numeric <- rasterize(shp, origStack, field = "vals")
+#' shpRas1_multiF <- rasterize(shp, origStack, field = c("other", "vals"))
+#' shpRas1_missingF <- rasterize(shp, origStack)
+#'
+#' shpRas2_character <- fastRasterize(shp, origStack, field = "other")
+#' shpRas2_numeric <- fastRasterize(shp, origStack, field = "vals")
+#' shpRas2_multiF <- fastRasterize(shp, origStack, field = c("other", "vals"))
+#' shpRas2_missingF <- fastRasterize(shp, origStack)
+#'
+#' testthat::expect_equal(shpRas1_character, shpRas2_character)
+#' testthat::expect_equal(shpRas1_numeric, shpRas2_numeric)
+#' testthat::expect_equal(shpRas1_multiF, shpRas2_multiF)
+#' testthat::expect_equal(shpRas1_missingF, shpRas2_missingF)
```

Comment:
please don't add tests to the examples! use the test-faster-raster.R file for tests.